src/main/scala/beam/agentsim/infrastructure/power/SitePowerManager.scala

```diff
@@ -28,46 +29,14 @@ class SitePowerManager(chargingNetworkMap: Map[Id[VehicleManager], ChargingNetwo
     */
   def requiredPowerInKWOverNextPlanningHorizon(tick: Int): Map[ChargingStation, PowerInKW] = {
     val plans = allChargingStations.par
-      .map { station =>
-        station -> observedPowerDemandInKW(tick, station.zone).getOrElse(estimatePowerDemandInKW(tick, station.zone))
-      }
+      .map(station => station -> temporaryLoadEstimate.getOrElse(station, 0.0))
       .seq
       .toMap
+    temporaryLoadEstimate.clear()
```

Comment:
This method should probably be renamed to useObservedLoad... or something like that, since it clears out the Map. Also, tick is no longer needed.